docs/docs/getting-started/01-intro.md

```diff
@@ -10,14 +10,14 @@ This guide will cover the basic steps to get started with the AICA System, from
 
 The AICA System is officially supported on Ubuntu and macOS for both amd64 and arm64 architectures.
 
-AICA Core runs in a Docker container based on Ubuntu 22.04. Thanks to containerization, the majority of the
+AICA Core runs in a Docker container based on Ubuntu 24.04. Thanks to containerization, the majority of the
 installation and usage steps in this guide will be the same for macOS or Ubuntu host platforms unless otherwise
 indicated.
 
 :::info
 
 There are some limitations with Docker on macOS compared to Docker on Ubuntu, which can affect advanced usage.
-For the best developer experience, use an Ubuntu 22.04 host machine.
+For the best developer experience, use an Ubuntu 24.04 host machine.
```

Comment:
```suggestion
For the best developer experience, use a recent Ubuntu distribution on your host machine.
```

I don't necessarily see why 24.04 would be the best. Having said that, I know that bleeding edge kernels (arch-based distros) do have some weird Docker issues. So I think it's ok to suggest Ubuntu, but not discourage the users from trying because they are for example on 20.04.

Might make more sense to mention Docker version?

Reply:
Good point, do we know anything about the Docker requirements @LouisBrunner ?

Reply:
Make sense for the OS. The only requirements for Docker is a somewhat recent version (one with buildx). Docker's version history is a bit tough to navigate but I imagine requiring Docker Engine v19 or later should be enough.